incubator/binding-mqtt-kafka/src/main/java/io/aklivity/zilla/runtime/binding/mqtt/kafka/internal/stream/MqttKafkaSessionFactory.java

```diff
@@ -2354,7 +2544,7 @@ private void onKafkaReset(
             assert acknowledge <= sequence;
             assert acknowledge >= delegate.initialAck;
 
-            delegate.initialAck = acknowledge;
+            delegate.initialAck = acknowledge - delegate.sessionPadding;
```

Comment:
This does not seem right.
Padding is used to increase the amount of reserved sent with DATA, and that is fully ack'd.
What is the rationale here? Perhaps trying to solve a different problem?